src/service/smarthome/smarthome.ts

```diff
@@ -60,7 +60,7 @@ export interface SmartHomeHandler<
   TRequest extends Api.SmartHomeV1Request,
   TResponse extends Api.SmartHomeV1Response,
   > {
-  (body: TRequest): TResponse | Promise<TResponse>
+  (body: TRequest, headers: object): TResponse | Promise<TResponse>
```

Comment:
Looking at other instances of the library, it looks like headers are of type [Headers](https://github.com/actions-on-google/actions-on-google-nodejs/blob/master/src/framework/framework.ts#L76), like in the [Actions SDK](https://github.com/actions-on-google/actions-on-google-nodejs/blob/master/src/service/actionssdk/conversation/conversation.ts#L18)